<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Analyzer</class>
 <widget class="QDialog" name="Analyzer">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1191</width>
    <height>1087</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Analyzer</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="3" column="0">
    <layout class="QHBoxLayout" name="localizations_layout"/>
   </item>
   <item row="6" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QPushButton" name="pbReset">
       <property name="text">
        <string>Reset</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="1" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <spacer name="horizontalSpacer_5">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLabel" name="lbCFRBoundsDetector">
       <property name="font">
        <font>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>CFR thresholding:</string>
       </property>
       <property name="textFormat">
        <enum>Qt::AutoText</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="checkCFRLowerThreshold">
       <property name="text">
        <string>Lower bound</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="checkCFRUpperThreshold">
       <property name="text">
        <string>Upper bound</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="lbCFRFilterThreshFactor">
       <property name="text">
        <string>Threshold factor</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="leCFRFilterThreshFactor"/>
     </item>
     <item>
      <widget class="QPushButton" name="pbCFRRunAutoThreshold">
       <property name="text">
        <string>Threshold</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pbCFRRunFilter">
       <property name="text">
        <string>Filter</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_6">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item row="0" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLabel" name="lbEFOPeakDetection">
       <property name="font">
        <font>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>EFO peak detection:</string>
       </property>
       <property name="textFormat">
        <enum>Qt::AutoText</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="checkEFOLowerThreshold">
       <property name="text">
        <string>Lower bound</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="checkEFOUpperThreshold">
       <property name="text">
        <string>Upper bound</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="lbEFOMedianFilterSupport">
       <property name="text">
        <string>Median filter support</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="leEFOMedianFilterSupport">
       <property name="sizePolicy">
        <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="lbEFOMinRelativeProminence">
       <property name="text">
        <string>Min peak relative prominence</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="leEFOMinRelativeProminence">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pbEFORunAutoPeakDetection">
       <property name="text">
        <string>Detect</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pbEFORunFilter">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>Filter</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item row="2" column="0">
    <layout class="QHBoxLayout" name="parameters_layout"/>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
